Doc/howto/logging-cookbook.rst

```diff
@@ -765,13 +765,71 @@ serialization.
-such as `Supervisor <http://supervisord.org/>`_. `Here
-<https://gist.github.com/vsajip/4b227eeec43817465ca835ca66f75e2b>`_ is a Gist which
-provides the bare-bones files to run the above functionality using Supervisor: you
-will need to change the ``/path/to/`` parts in the Gist to reflect the actual paths you
-want to use.
-
+To run a logging listener in production, you may need to use a
+process-management tool such as `Supervisor <http://supervisord.org/>`_. `Here
+<https://gist.github.com/vsajip/4b227eeec43817465ca835ca66f75e2b>`_ is a Gist
+which provides the bare-bones files to run the above functionality using
+Supervisor. It consists of the following files:
+
++-------------------------+----------------------------------------------------+
+| File                    | Purpose                                            |
++=========================+====================================================+
+| :file:`prepare.sh`      | A Bash script to prepare the environment for       |
+|                         | testing                                            |
++-------------------------+----------------------------------------------------+
+| :file:`supervisor.conf` | The Supervisor configuration file, which has       |
+|                         | entries for the listener and a multi-process web   |
+|                         | application                                        |
++-------------------------+----------------------------------------------------+
+| :file:`ensure_app.sh`   | A Bash script to ensure that Supervisor is running |
+|                         | with the above configuration                       |
++-------------------------+----------------------------------------------------+
+| :file:`log_listener.py` | The socket listener program which receives log     |
+|                         | events and logs them to a file                     |
++-------------------------+----------------------------------------------------+
+| :file:`main.py`         | A simple web application which does logging via a  |
+|                         | socket connected to the listener                   |
++-------------------------+----------------------------------------------------+
+| :file:`webapp.json`     | A JSON configuration file for the web application  |
++-------------------------+----------------------------------------------------+
+| :file:`client.py`       | A Python script to exercise the web application    |
++-------------------------+----------------------------------------------------+
+
+The web application uses `Gunicorn <https://gunicorn.org/>`_, which is a
+popular web application server which starts multiple worker processes to handle
+requests. The whole setup shows how the workers can write to the same log file
+without treading on each other's toes --- they go through the socket listener.
+
+To test these files, do the following in a POSIX environment:
+
+1. Download `the Gist
```

Comment:
I think this can be [autonumbered lists](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#lists-and-quote-like-blocks)

Reply:
Yes, autonumbered lists are generally preferable for longer ordered lists, as they avoid tedious effort, commit noise and the chance of mistakes when items or added, removed or moved.

```suggestion
#. Download `the Gist
```

Reply:
For some reason, when I used `#`, it didn't autonumber on my machine ... no idea why. Rendered output showed as `#`!

Reply:
Hmm, assuming you built from the same source as here (which autonumbers just fine on my machine with the latest docs theme and Sphinx 4.5.0, same as on the CIs and build server), my guess would be some issue with the Sphinx version you were using.

Reply:
Possibly, though it's a very recent version as set up using `make venv` in the `Docs` folder :confused: